flask_githubapp/core.py

```diff
@@ -184,7 +184,15 @@ def _flask_view_func(self):
                                            400)
             abort(error_response)
 
-        event = request.headers['X-GitHub-Event']
+        event_header = 'X-GitHub-Event'
```

Comment:
I wonder if it's not possible to simply use `.get('X-GitHub-Event', None)` and handle the case that's `None`
It's more a pet peeve than anything else, the check is 👌

Reply:
Ah I like this a lot thanks. I'm going to omit the optional second arg to `.get` (it defaults to `None`) because that's the style I see most often in Python.